book/embeds.md

```diff
@@ -2196,10 +2196,10 @@ network, preventing private data from leaking.
 
 These isolation and sandboxing features may seem "straightforward", in
 the same sense that the browser thread we added in [Chapter
-13](scheduling.md) is "straightforward". In practice, the many browser
+12](scheduling.md) is "straightforward". In practice, the many browser
 APIs mean the implementation is full of subtleties and ends up being
 extremely complex. Chromium, for example, took many years to ship the
-first implementation of site isolation.
+first implementation of *site isolation*.
 
 [site-isolation]: https://www.chromium.org/Home/chromium-security/site-isolation/
```

Comment:
Is this link used anywhere? Instead of italicizing "site isolation" it would I think look better to link it.

Reply:
This is defining a term used again later, which needs italics. I added the link (there is another link to it earlier in the chapter)